src/lib.rs

```diff
@@ -141,17 +140,12 @@ pub mod url {
 }
 
 /// Status Codes
-pub mod status {
-    pub use hyper::status::StatusCode as Status;
-    pub use hyper::status::StatusCode::*;
-    pub use hyper::status::StatusClass;
-}
+pub use http::status as status;
+pub use http::status::StatusCode;
```

Comment:
Why not use

    pub use http::status::StatusCode as Status

and not import `http::status` at all? This would also spare us some renaming¹ in lots of other places.

As far as I can tell, `http::status` basically never used except for some occurences of `status::StatusCode`—which already is imported as `StatusCode` right now.

¹ Not that it really would spare us renaming, since the names have all been changed to ALL_CAPS, but still …

Reply:
Umm... Like hyper? :D Seems like a good idea

Reply:
Ok hyper just exports StatusCode, I think we should also do that.